### DISCUSSION TITLE
How to download all subscribed subreddit without list them all?

### DISCUSSION THREAD

**Discussion by xXTheCoolGamerXx** [Serene-Arc/bulk-downloader-for-reddit#529] [Q&A]
How to download all subscribed subreddit without list them all?

**Comment by Serene-Arc** [comment]
That's currently not possible.

**Reply by xXTheCoolGamerXx** [reply]
Why with the exe we can and the new one we cant? Its said its exactly the same thing, just coded in python.

**Reply by Serene-Arc** [reply]
It's not, I rewrote the tool from scratch trying to keep feature parity. If this is a feature I missed or you want it, you are welcome to open up a feature request.
